### ISSUE TITLE
[Quest][Blade´s Edge Mountains] The Smallest Creatures

### ISSUE THREAD

**Issue by Dirtysprite420** [Atlantiss/NetherwingBugtracker#7584]
**Description**:  When you use the Kodohide Drum at the Marmot Den , you just get into a stealth mode with your character.
**Expected Behaviour**: You should be able to play as a Marmot to poison the Bloodmaul Brews.

**Comment by therealCrysis** [comment]
Still bugged.

**Comment by dumogin** [comment]
This quest still doesn't work correctly.

At first I was in this invisibility zone without pet. I opened a GM ticket and tried the different instructions provided by the GM (clearing cache, targeting different marmots). I also tried with my pet summoned and not and with and without interface customization. 
![Wow_qoCuJNXGsx](https://user-images.githubusercontent.com/1774534/101617452-ec1ad000-3a10-11eb-93f8-6e97b5c597ad.jpg)

The GM then told me to wait for a server restart. The first try worked but my marmot was under the world as seen in the screenshot below. 
![Wow_BpIQjLCltW](https://user-images.githubusercontent.com/1774534/101617457-ed4bfd00-3a10-11eb-8de8-465b41568f5b.jpg)

Subsequent tries lead to the result provided in the first screenshot. I even tried killing the marmots this didn't help either. Poor things. 

I hope this information helps you with fixing this bug.

**Comment by dumogin** [comment]
Alright I have a small update I tried it again and it worked. I don't know what I did differently. Things I did before one try failed and one try was a success:
- Killing all marmots
- I targeted the marmot in the cave
- I stood closer to the cave than in my other tries.

I don't think these things are related since they weren't the case on the try where I fell through the world. I was probably just lucky and got the right one.
